### PR TITLE
Minor refactoring

### PR DESCRIPTION
**Pull request by swift-kim**
- Simplify logic related to the x64 shell.
- Make the `clipboard` namespace private to `platform_channel.cc`. No actual code change.

### CODE REVIEW

File: shell/platform/tizen/channels/platform_channel_stub.cc [hunk 1]
```diff
@@ -7,24 +7,8 @@
...
-    const MethodCall<rapidjson::Document>& call,
-    std::unique_ptr<MethodResult<rapidjson::Document>> result) {}
-
-namespace clipboard {
-
-void GetData(const MethodCall<rapidjson::Document>& call,
-             std::unique_ptr<MethodResult<rapidjson::Document>> result) {}
-
-void SetData(const MethodCall<rapidjson::Document>& call,
-             std::unique_ptr<MethodResult<rapidjson::Document>> result) {}
```
Comment (bbrto21) [inline shell/platform/tizen/channels/platform_channel_stub.cc:26]: Does excluding these functions from stub mean that there are no plans to test any more?

Comment (swift-kim) [reply shell/platform/tizen/channels/platform_channel_stub.cc:26]: Well, I think we can move the namespace `clipboard` into a separate header file if we'd like to test the methods later.

Comment (bwikbs) [reply shell/platform/tizen/channels/platform_channel_stub.cc:26]: Currently,I am  trying to make a unit test of a platform channel.
For this purpose, I have a plan to change these functions to a protected method of the platform channel.
What do you think of it?

Comment (swift-kim) [reply shell/platform/tizen/channels/platform_channel_stub.cc:26]: I'm okay with it although I'm not the original author of the functions. Actually the current platform_channel implementation is quite messy and I had a plan to refactor it before we write unit tests for it.

Comment (bwikbs) [reply shell/platform/tizen/channels/platform_channel_stub.cc:26]: Oh if you already have plans I'll do something else.

File: shell/platform/tizen/channels/settings_channel.h [hunk 2]
```diff
@@ -27,11 +23,13 @@ class SettingsChannel {
   virtual ~SettingsChannel();
 
  private:
+#ifndef __X64_SHELL__
```
Comment (bwikbs) [inline shell/platform/tizen/channels/settings_channel.h:26]: Are you preparing to refactor the setting channel too? If so, it doesn't matter... but ,for testing purposes, Interface differences would be minimized as possible.

Comment (swift-kim) [reply shell/platform/tizen/channels/settings_channel.h:26]: No, I'm not. The settings channel isn't implemented by the x64 embedder, then how do you run tests for unimplemented methods using the x64 shell?

Comment (bwikbs) [reply shell/platform/tizen/channels/settings_channel.h:26]: Using ``MOCK_METHOD``?
I'm not testing the actual operation of the function, I'm just trying to test the interface.

Comment (swift-kim) [reply shell/platform/tizen/channels/settings_channel.h:26]: I think I somewhat understand what you're trying to do, but I still doubt if you really want to test these private methods. These methods are only there for the device embedder implementation (settings_channel.cc), and not used by the desktop embedder (settings_channel_stub.cc). You may later exclude some methods from #ifdef after adding some desktop embedder-side implementation of `SettingsChannel`.

Comment (bwikbs) [reply shell/platform/tizen/channels/settings_channel.h:26]: Right, I agree with what you're talking about. 
I guess further discussion is not relevant this PR. :smile: 
Let's discuss next individual unit test PR.

**Review by bwikbs — APPROVED** [review]
Thanks for your working!!

File: shell/platform/tizen/channels/settings_channel.h (hunk 2 above)
Comment (swift-kim) [reply shell/platform/tizen/channels/settings_channel.h:26]: Okay, let's talk about the matter later. Thanks!

**Review by bbrto21 — APPROVED** [review]
LGTM

**Pull request merged**